packages/ptokens-assets-evm/test/ptokens-utils-evm.test.ts

```diff
@@ -1,16 +1,31 @@
 import BigNumber from 'bignumber.js'
 import { stringUtils } from 'ptokens-helpers'
-import { Web3 } from 'web3'
+import { Log } from 'viem'
+import { NetworkId } from 'ptokens-constants'
+import { polygon } from 'viem/chains'
 
 import * as utils from '../src/lib'
-
-import abi from './utils/exampleContractABI'
 import logs from './utils/logs.json'
-
-const TEST_CONTRACT_ADDRESS = '0x15FA11dFB23eae46Fda69fB6A148f41677B4a090'
-const TEST_ETH_PRIVATE_KEY = '422c874bed50b69add046296530dc580f8e2e253879d98d66023b7897ab15742'
+import { publicClient } from './utils/viem-clients'
 
 describe('ethereum utilities', () => {
+  test('Should return a Viem chain', () => {
+    require('ptokens-constants').INTERIM_NETWORK_ID = NetworkId.PolygonMainnet
```

Comment:
You see there is already INTERIM_NETWORK_ID. Use it in the swap builder for the default interim net id.